components/sysutils/tmux/Makefile

```diff
@@ -11,20 +11,21 @@
 #
 # Copyright 2011, 2013, EveryCity Ltd. All rights reserved.
 # Copyright 2019, Michal Nowak
+# Copyright 2019, Solene Rapenne
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		tmux
-COMPONENT_VERSION=	2.9
+COMPONENT_VERSION=	3.0
 COMPONENT_FMRI=		terminal/tmux
```

Comment:
There is a special treatment for version numbers pkg cannot cope with:
COMPONENT_VERSION=   3.0a
IPS_COMPONENT_VERSION= 3.0.1
(See openssl for another example)